packages/local_auth/local_auth_ios/ios/Classes/FLTLocalAuthPlugin.m

```diff
@@ -216,41 +220,35 @@ - (void)handleAuthReplyWithSuccess:(BOOL)success
     result(@YES);
   } else {
     switch (error.code) {
+      case LAErrorSystemCancel:
+        if ([arguments[@"stickyAuth"] boolValue]) {
+            self->_lastCallArgs = arguments;
+            self->_lastResult = result;
+        } else {
+            result(@NO);
+        }
+        return;
       case LAErrorPasscodeNotSet:
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-      // TODO(stuartmorgan): Remove the pragma and s/TouchID/Biometry/ in these constants when
-      // iOS 10 support is dropped. The values are the same, only the names have changed.
```

Comment:
is ios 10 dropped?

Reply:
Yes (?), I was under the impression minimum ios version is 11.0 now

Reply:
hmmm, i thought it was ios 9 but i could remember it wrongly. Can you double check other plugins?

Reply:
On the flutter docs it says minimum version is 11 https://docs.flutter.dev/deployment/ios#:~:text=The%20minimum%20iOS%20version%20that,to%20the%20highest%20required%20version.

Reply:
that's the current flutter version (people could be using older flutter)

Reply:
quick search and found [this](https://github.com/flutter/flutter/issues/101965) - we are still at ios 9 here.

Reply:
Huan is right, this version of the plugin should continue to work on iOS 9 until the minimum version is bumped to 11.  The tests are passing because PRs only run on Flutter master, which is auto-migrating to iOS 11 so there's no issue.  I think that would even happen in stable too, which only gets run on post-submit.

Trying to think how to force the app to run against the iOS 9 SDK in CI.  I guess we just haven't run into issues related to this yet... cc @stuartmorgan

Reply:
> Trying to think how to force the app to run against the iOS 9 SDK in CI.

Yeah, that's a problem given the auto-migrate logic. I would suggest someone just do a mass-update of all the iOS implementation packages to Flutter 3.3 and iOS 11 now (or maybe 3.3 first, and then iOS 11 as a second pass if that's non-trivial due to dead code removal).

That's the approach I've adopted for our old-Flutter-version support: when we update the N-2 stable version in CI, I mass-update all plugins to at least that minimum version, because we're no longer getting any automated testing that we aren't accidentally breaking them. (That kind of update doesn't cause any churn for users, because people still using old Flutter versions will just not be given the updates they can't use due to the resolver constraint on the Flutter version.)